### PR TITLE
Add guide for CodiMD

### PR DESCRIPTION
**Pull request by Stunkymonkey**
I am confused with the merged #200 and open #218. I am not able to append commits to this.
I hope this is the right way :confused: 

All the requested changes in #218 are done.

### CODE REVIEW

File: source/guide_codimd.rst [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+  [...]
+  [isabell@stardust codimd]$
+
+
+Remove UglifyJS
+---------------
+
+The UglifyJS plugin freezes the compilation step, because it tries to compile everything in parallel and then reaches the maximum memory-limit/user of uberspace. And since it's apparently not necessary, we'll just remove it.
+
+Open ``webpack.production.js`` and comment out this line (currently line 6):
```
Comment (SalocinHB) [inline source/guide_codimd.rst]: Please explain why

File: source/guide_codimd.rst [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+
+So that it looks like this:
+
+.. code-block:: javascript
+  :emphasize-lines: 1
+
+  //var ParallelUglifyPlugin = require('webpack-parallel-uglify-plugin')
+
+
+And comment out this block (currently line 15 - 26):
```
Comment (SalocinHB) [inline source/guide_codimd.rst]: Please explain why

Comment (SalocinHB) [reply source/guide_codimd.rst]: Doesn't work
```
[testnico@machholz codimd]$ npm run build
npm WARN lifecycle The node binary used for scripts is /bin/node but npm is using /opt/nodejs8/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> CodiMD@1.2.1 build /home/testnico/codimd
> webpack --config webpack.production.js --progress --colors --bail

/home/testnico/codimd/webpack.production.js:15
    /*new ParallelUglifyPlugin({
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid or unexpected token
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at module.exports (/home/testnico/codimd/node_modules/webpack/bin/convert-argv.js:80:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! CodiMD@1.2.1 build: `webpack --config webpack.production.js --progress --colors --bail`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the CodiMD@1.2.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/testnico/.npm/_logs/2018-10-24T21_08_55_763Z-debug.log
```

File: source/guide_codimd.rst (hunk 1 above)
Comment (Stunkymonkey) [reply source/guide_codimd.rst]: compiling everything in parallel uses more ram.
the available ram per user at uberspace is around ~1,5GB.
the parallel compilation needs more than 2GB.
the sequential compilation needs less than 1,5GB.
that is why we will use the sequential one.
CodiMD itself uses parallel by default, but we can disable it via uncommenting `UglifyJS`

File: source/guide_codimd.rst (hunk 2 above)
Comment (Stunkymonkey) [reply source/guide_codimd.rst]: can you paste the output of the first 35 lines of your `webpack.production.js`?

**Comment by Stunkymonkey** [comment]
i made a new commit, you did not reply to.
the one where i added the editing of`.sequelizerc` (maybe this was the problem)
on [codi.uber.space](codi.uber.space) i tested it again, and it worked.
@SalocinHB can you please try this guide again.

**Comment by Stunkymonkey** [comment]
thank you for everything
and I'm sorry to keept you waiting.

**Pull request merged**